packages/mdc-linear-progress/foundation.ts

```diff
@@ -91,6 +102,17 @@ export class MDCLinearProgressFoundation extends MDCFoundation<MDCLinearProgress
 
   setReverse(isReversed: boolean) {
     this.isReversed_ = isReversed;
+
+    if (!this.isDeterminate_) {
+      // reset scaling animation timer that was started when indeterminate
+      // class was introduced, as to keep it in sync with the reversed
+      // translation timer that's about to start from adding/removing reversed
+      // class
```

Comment:
Also would reword to something like:

Adding/removing REVERSED_CLASS starts a `translate` animation, while adding INDETERMINATE_CLASS starts a `scale` animation. 
Here, we reset the `scale` animation in order to keep it in sync with the new `translate` animation that will start from adding/removing REVERSED_CLASS below.